androidApp/src/main/java/com/pulselink/ui/screens/ExtensionsStoreScreen.kt

```
@@ -0,0 +1,343 @@
+                }
+
+                item {
+                    ExtensionCTA()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun StoreItemCard(feature: FeatureToggle, premiumActive: Boolean) {
```

detekt (1.23.8)

[warning] The function StoreItemCard appears to be too complex based on Cognitive Complexity (complexity: 33). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)